CHANGELOG.md

```diff
@@ -2849,7 +2857,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
 - Added the ability to restore accounts from seed words.
 
-[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.12.2...HEAD
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.12.4...HEAD
+[10.12.4]: https://github.com/MetaMask/metamask-extension/compare/v10.12.2...v10.12.4
```

Comment:
Should this be 10.12.3?

Reply:
Fixed, thanks!

Reply:
Oh... I guess the Markdown doc expects us to list something here though... hmm. Maybe we should create this tag then.

Reply:
yes tests were failing without that tag!